tools/crm_resource_runtime.c

```diff
@@ -1823,7 +1839,7 @@ cli_resource_execute(pe_resource_t *rsc, const char *requested_name,
                      const char *rsc_action, GHashTable *override_hash,
                      int timeout_ms, cib_t * cib, pe_working_set_t *data_set)
 {
-    int rc = pcmk_ok;
+    int rc = pcmk_rc_ok;
```

Comment:
This would be the other function returning crm_exit_t. rc is used in two different senses in this function so probably worth separating that into two variables